noteable/datasources.py

```diff
@@ -241,9 +254,18 @@ def pre_process_dict(the_dict: Dict[str, Any]) -> None:
 DUCKDB_LOCATION = "duckdb:///:memory:"
 
 
-def bootstrap_duckdb(registry: ConnectionRegistry):
-    registry.factory_and_register(
+def local_duckdb_bootstrapper() -> Connection:
+    """Return the noteable.sql.connection.Connection to use for local memory DuckDB."""
+    return Connection(
         sql_cell_handle=LOCAL_DB_CONN_HANDLE,
         human_name=LOCAL_DB_CONN_NAME,
         connection_url=DUCKDB_LOCATION,
     )
+
+
+def queue_bootstrap_duckdb(registry: ConnectionRegistry):
```

Comment:
This is a toplevel function and not inlined where called here in datasources package 'cause also useful for the test suite to call directly.